crates/rethnet_eth/src/remote/client.rs

```diff
@@ -1088,7 +1202,7 @@ mod tests {
         }
 
         #[tokio::test]
-        async fn get_block_with_transaction_data() {
+        async fn get_block_with_transaction_data_cached() {
```

Comment:
Was this just attempting to reduce the number of tests (and thus queries) by merging the tests for `get_block_with_transaction_data` and caching?

Reply:
Yes, renaming is follow up from previous PR. There are a bunch of other block number with transaction data tests too